src/Redux/companiesReducer.js

```diff
@@ -0,0 +1,37 @@
+export const FETCH_COMPANIES = 'FETCH_COMPANIES';
```

Comment:
- [Optional] It's always good to scope our actions name based on the feature they represent. This will really help you when scaling your app in the future.